config.go

```diff
@@ -114,7 +114,7 @@ var defaultConfig = &Config{
 	ActiveExtension:      "",
 	KeyboardMacros:       []KeyboardMacro{},
 	DisplayRotation:      "270",
-	KeyboardLayout:       "en_US",
+	KeyboardLayout:       "en-US",
```

Comment:
... once and for all fixing this for old and new devices with configs set or not set... so it should always work now (I tested by a factory reset and found there were still issues)